src/axom/core/tests/core_numeric_array.cpp

```diff
@@ -276,7 +272,8 @@ TEST(primal_numeric_array, clamping)
   // NOTE: AXOM_DEBUG is disabled in release mode, so this test will only fail
   // in
   // debug mode
```

Comment:
Minor: I know you didn't change this, but would you mind cleaning up the awkward spacing in this comment?
```suggestion
  // NOTE: AXOM_DEBUG is disabled in release mode, so this test will only fail in debug mode
```

Reply:
Sure!